include/oneapi/dpl/pstl/memory_impl.h

```diff
@@ -41,7 +41,7 @@ __brick_uninitialized_move(_ForwardIterator __first, _ForwardIterator __last, _O
     using _ValueType = typename ::std::iterator_traits<_OutputIterator>::value_type;
     for (; __first != __last; ++__first, ++__result)
     {
-        ::new (::std::addressof(*__result)) _ValueType(::std::move(*__first));
+        ::new (std::addressof(*__result)) _ValueType(std::move(*__first));
```

Comment:
Should we really touch this and other unrelated parts of this patch only to remove the leading `::` from `std`?

Reply:
Yes, I think so. I hope our colleagues will also write their opinion about this.
@rarutyun ?

Reply:
I would only touch the pieces of the code that have "true" modifications. Probably I would consider to change something that is physically close in the code but also has some logical connection. In other words, I agree with Julian's concern and would not modify these pieces that only remove `::` before `std` (

Reply:
OK, these extra changes has been removed.